setup.py

```diff
@@ -38,7 +38,7 @@
 )
 
 install_requires = [
-    'web3>=4.0.0,<5.0.0',
```

Comment:
No CI set up on this mini project unfortunately. Were you able to test it out?

Since it doesn't look like any code changes are needed, it would be ideal to continue to support web3 v4 as well.

Reply:
I ran
```
make lint && make build-docs && pytest tests
```
and everything looked good, but I'll attach test evidence 👍 And yeah, I agree if there's no need to drop v4 support we should keep it in there.

Reply:
Hah, yeah, this is a *very* alpha project. If you take a look at "tests", you'll see what I mean.

Maybe the best solution is actually to break the dependency from web3.py on `ethtoken` during testing. The `web3` doctest only uses the ABI, so we could just copy that ABI directly into the web3 doctest for now. That would simplify our lives and remove a dependency on a not-really-maintained project.

Reply:
Notably, I think that upcoming web3 v5 changes *will* break ethtoken, so I'm not sure I want to mark it as supported yet. (`ConciseContract` will go away, for example)

Reply:
One simple test is better than no tests though :)

But yeah, ideally we wouldn't be creating a circular dependency... I can look into that and see what that would entail 🤔

Reply:
@carver here's the PR:

https://github.com/ethereum/web3.py/pull/1202